lib/build-tokens.js

```diff
@@ -40,6 +58,10 @@ async function buildTokensCommand(commandArgs) {
     source: tokensSource
       ? [`${tokensSource}/core/**/*.json`, `${tokensSource}/core/**/*.toml`]
       : [],
+    preprocessors: ['tokens-studio'],
+    expand: {
+      typesMap: (await getTokensStudioStyleDictionaryTransforms()).expandTypesMap,
+    },
```

Comment:
[inform] Largely a no-op change right now (i.e., except for some minor formatting in the output, e.g. including leading zeros in `0.625rem`), but these changes should make it possible to support/define tokens exported from the Figma plugin Token Studio.